### ISSUE TITLE
STORY: TFGrid Dashboard v3.7.0

### ISSUE THREAD

**Issue by renauter** [threefoldtech/home#1215]
## Why a all-in-one TFGrid Dashboard ?

Following the idea of simplifying the user experience we want to have a more user-friendly TFChain Portal 
Actually we have all kind of different services which are relevant for the user (e.g. Playground / Grid Explorer / ...) and not obviously accessible from the portal, that's why we want to unify them
We decided to create a new product called TFGrid Dashboard to replace the actual portal

## Story items

- TFGrid Dashboard [repo](https://github.com/threefoldtech/tfgrid_dashboard) and [project](https://github.com/threefoldtech/tfgrid_dashboard/projects/1)
- Potentially affected repos
  - ~[Playground / Weblets](https://github.com/threefoldtech/grid_weblets)~
  - [Grid Explorer](https://github.com/threefoldtech/nodes-explorer) 
- Technical issues
  - [ ] define a uniform design for everything (use same look & feel for all components)
  - [x] Integrate Grid Explorer natively in portal
  - [x] Integrate the DAO of tfchain
  - [x] Integrate TFHub
  - [ ] ~Integrate Playground / Weblets in Portal (show as sort of marketplace)~ Different technologies mix, different UI frameworks used, it needs to be either fully rebuilt with threefold styling or another dedicated system-

## Future consideration

- The TFGrid Dashboard should also be able to directly connect using the future TFConnect Web Wallet (#1210) and get rid of Polkadot browser extension dependency 
- Playground integration was discarded for first version of dashboard but should be considered for next versions
- Make sure Playground will use TFConnect Web Wallet (no longer the custom profile manager, we should use TFConnect Web Wallet for everything)

## Open questions

Shall we go first for a soft integration as suggested [here](https://github.com/threefoldtech/home/issues/1215#issuecomment-1102274105) and rebuild a new product after ?
Question was answered by creating new dashboard instead of keeping portal

**Comment by xmonader** [comment]
Admin portal and Explorer are using two different technologies and the playground is using another one. The best route is unifying the look and feel by using a company library in all products.

Playground, is also having an issue with custom css and the webcomponents which won't allow the same look&feel.

Integration with TF Connect should be out of the scope of this until it's ready.

**Comment by sasha-astiadi** [comment]
issue merged into https://github.com/threefoldtech/home/issues/1221, please comment and continue discussion on this issue

**Comment by renauter** [comment]
Re-opened this issue because it seems to be clearer to handle the unified TFChain Portal separately

**Comment by renauter** [comment]
See @sabrinasadik comments on #1195 to complete the story

**Comment by sabrinasadik** [comment]
# Portal
- If your wallet is not connected to the portal, you should be able to see the explorer, which should be integrated into the portal. Once you are connected, you're able to see all other functionalities as well.
- Connect through the TF app instead of Polkadot extension (as soon as Jimber is ready).
- Signing transactions should be approved through the app with a pop-up (dependant on the above).
- Integrate GETTFT into the portal so people can directly buy tokens through the portal since their wallet is already in there (not possible according to @xmonader, but we can add a link to the portal which brings you to gettft).
- Optimise https://alpha.minting.tfchain.grid.tf/ and make sure people can see their minting history and details when clicking on a node on the farmer section of the portal. 
- Eventually, all of this can be integrated into the TF Connect App (webview of the portal).

**Comment by xmonader** [comment]
The goal is to have a unified look and feel not merging tons of different technologies/languages and code into the same project without a real benefit -except for the added complexity and scope creep-. 


For me integration of portal and explorer is kinda nice, having the minting integrated too is a really good idea. For other things I don't agree.

##  GET TFT 
Tech: python, electrum, bitcoin,  js-sdk, depends on v2 explorer, tf connect, configmanager for database, kubernetes for infrastructure, mercurio payment gateway. vue2, vuetify, nginx for frontened

Get TFT is an example of things I don't agree they need to be redone what makes sense is updating the UI to match the brandbook

## Playground
Playground itself is done in vue as a showcase for embeddability of weblets, and weblets are done as web components in svelte, but it should be just a playground and limited to that

Tech: svelte, webcomponents, typescript, Bulma, vue2

Weblets are meant to be as plain as possible for reusability (e.g in a playground on in an interactive documentation.. etc), what could be a better option is having a fully different project with its pieces more controlled and integrated, unlike the playground, because weblets are meant to be decoupled as much as possible while a well-defined experience needs to be really integrated.

**Comment by renauter** [comment]
Ok so, would it be possible to have a uniformed design pattern for Portal, GetTFT and Playground ?
At least background color, font style/size/color, for a better user experience

**Comment by xmonader** [comment]
definitely :) 

> Ok so, would it be possible to have a uniformed design pattern for Portal, GetTFT and Playground ? At least background color, font style/size/color, for a better user experience

Definitely

**Comment by renauter** [comment]
Great! Very important to have the feeling of everything looking the same for a unified experience, even if don t (still) belong to the same product

**Comment by robvanmieghem** [comment]
Given a micro-service architecture, the backends of the different functionalities should not be merged. The backend technology should not dictate how the frontends look like and there can be a standalone one and one in a unified portal.

**Comment by sasha-astiadi** [comment]
@rkhamis please create a new issue amd tag this one as history

**Comment by xmonader** [comment]
> @rkhamis please create a new issue amd tag this one as history

@sasha-astiadi  Why? This issue is perfectly fine. Also the story is in verification why would we redo all that again?

**Comment by renauter** [comment]
Maybe the best would be editing this one to better fit what was done on dashboard
Let me do that
Ok? @xmonader @sasha-astiadi
